doc/source/examples/extended_examples/index.rst

```diff
@@ -1,9 +1,23 @@
 .. _ref_extended_examples:
 
+.. toctree::
+    :hidden:
+    :maxdepth: 3
+
+    Cart-Pole Simulation <http://cartpole.mapdl.docs.pyansys.com/ml-rl-notebook.html>
+    ex_01-gmsh_example/ex_01-gmsh_example
+    Python_UPF/python_upf_examples
+    Krylov/krylov_example
+    dash/dash_example
+    executable/executable
+
+
+.. extended_text_start
```

Comment:
Pinging @Revathyvenugopal162 

This part is before the ``extended_text_start`` tag, so it should not be included in the ``include`` in ``examples/index.rst`` (line 39).

So the should not be so many examples on the left side.

<img width="385" alt="image" src="https://github.com/ansys/pymapdl/assets/28149841/7e655acd-2167-45b9-95f7-7e2315665444">

Reply:
It seems to me this is a bug? ... The toctree should not be rendered when using the ``include`` in ``examples/index.rst`` (because of the ``:start_after:``), but it should be when rendering ``extended_examples/index``. But if the toctree is present, it is always rendered regardless the ``:start_after:``.

Reply:
https://github.com/sphinx-doc/sphinx/issues/701#issuecomment-659062227